lint/src/test/java/com/firebaseui/lint/NonGlobalIdDetectorTest.kt

```diff
@@ -0,0 +1,43 @@
+package com.firebaseui.lint;
```

Comment:
Naughty! 😄 Semicolons are so from the 70s.

Reply:
Oh this happened when converting the original file from java to kotlin. Will fix.